samcli/commands/list/cli_common/options.py

```diff
@@ -0,0 +1,34 @@
+"""
+Common CLI options shared by various commands
+"""
+
+import click
+
+
+def stack_name_click_option():
+    return click.option(
+        "--stack-name",
+        help=(
+            "Name of corresponding deployed stack.(Not including "
+            "a stack name will only show local resources defined "
+            "in the template.) "
+        ),
+        type=click.STRING,
+    )
+
+
+def stack_name_option(f):
+    return stack_name_click_option()(f)
+
+
+def output_click_option():
+    return click.option(
```

Comment:
we should start to standardize this across commands.

Reply:
Agreed. Probably best to do in a follow-up PR to keep the scope of this one reasonable.